elftools/dwarf/die.py

```diff
@@ -237,11 +237,17 @@ def _parse_DIE(self):
 
         # Guided by the attributes listed in the abbreviation declaration, parse
         # values from the stream.
-        for name, form in abbrev_decl.iter_attr_specs():
+        for spec in abbrev_decl['attr_spec']:
+            form = spec.form
+            name = spec.name
             attr_offset = self.stream.tell()
-            raw_value = struct_parse(structs.Dwarf_dw_form[form], self.stream)
-
-            value = self._translate_attr_value(form, raw_value)
+            # Special case here:
```

Comment:
Comment here could use a little elaboration on why it's a special case.

Reply:
E. g.: 
```
# Special case here: the attribute value is stored in the attribute definition
# in the abbreviation spec, not in the DIE
```

Reply:
Thank you, I added this to the commit introducting support for implicit consts.